libs/storage/include/storage/file_object.hpp

```diff
@@ -372,6 +372,7 @@ void FileObject<S>::ReadWriteHelper(uint8_t const *bytes, uint64_t num, Action a
     switch (action)
     {
     case Action::READ:
+      // NOLINTNEXTLINE
       memcpy((uint8_t *)(bytes + bytes_offset), block_being_written.data + byte_index,
```

Comment:
What is not linger here

Reply:
It wants a `const_cast` and I chickened out.